source/frontend/src/features/mapSideBar/disposition/form/DispositionForm.tsx

```diff
@@ -55,8 +87,28 @@ const DispositionForm = React.forwardRef<FormikProps<DispositionFormModel>, IDis
             <>
               <Container>
                 <Section header="Project">
-                  <SectionField label="Ministry project"></SectionField>
-                  <SectionField label="Product"></SectionField>
+                  <SectionField label="Ministry project">
+                    <ProjectSelector
+                      field="project"
+                      onChange={(vals: IAutocompletePrediction[]) => {
+                        onMinistryProjectSelected(vals);
+                        if (vals.length === 0) {
+                          formikProps.setFieldValue('project', 0);
```

Comment:
Is this correct? Or should this be product?

Reply:
it should be product. updated.